src/nifake/templates/session.py/datetime_wrappers.py.mako

```diff
@@ -0,0 +1,27 @@
+<%page args="f, config"/>\
+    def get_cal_date_and_time(self, cal_type):
+        '''get_cal_date_and_time
+
+        Returns the date and time of the last calibration performed.
```

Comment:
I would simplify this and not make it NI-DMM specific.

Reply:
How? This function is very NI-DMM specific. No one else returns all these parameters.

Reply:
Okay, I''ll make it different from the nidmm function.